src/main/java/sirius/biz/tenants/UserAccountData.java

```diff
@@ -337,6 +354,27 @@ public boolean canSendGeneratedPassword() {
         return Strings.isFilled(email) && userObject.isUnique(UserAccount.USER_ACCOUNT_DATA.inner(EMAIL), email);
     }
 
+    /**
+     * Returns the parsed user preferences.
+     * <p>
+     * Use {@link UserAccount#readPreference(String)} to access the preferences.
+     *
+     * @return the parsed user preferences as map
+     */
+    public Map<String, Object> getUserPreferences() {
```

Comment:
rename to fetchUserPreferences / parseUserPreferences, since this is no direct getter?